src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -95,6 +95,19 @@ export const generateSigningKeyPair = (): Future<Error, SigningKeyPair> => Futur
 export const getPublicSigningKeyFromPrivate = (privateSigningKey: PrivateKey<Uint8Array>) =>
     Future.tryF(() => RecryptApi.computeEd25519PublicKey(privateSigningKey));
 
+/**
+ * Augments UserPrivateKey and return both the augmented private key and the augmentation factor.
+ * @param userPrivateKey current private key to be augmented
+ */
+export const rotateUsersPrivateKey = (userPrivateKey: Uint8Array): Future<Error, {newPrivateKey: Uint8Array; augmentationFactor: Uint8Array}> => {
+    return generateKeyPair().map(({privateKey}) => {
+        return {
+            newPrivateKey: Recrypt.subtractPrivateKeys(userPrivateKey, privateKey),
+            augmentationFactor: privateKey,
+        };
+    });
```

Comment:
Very nitpicking, but you can avoid the second return by doing

```
    return generateKeyPair().map(({privateKey}) => ({
        newPrivateKey: Recrypt.subtractPrivateKeys(userPrivateKey, privateKey),
        augmentationFactor: privateKey,
    }));
```

Reply:
done